tdp/core/dag.py

```diff
@@ -23,6 +23,19 @@
 
 logger = logging.getLogger("tdp").getChild("dag")
 
+SERVICES_ORDER = {
```

Comment:
Rename it `SERVICES_PRIORITIES`, this order do not change the DAG order, so it is more about priorities.